### PR TITLE
Fix some bugs

### PR DESCRIPTION
**Pull request by Matthww**
And added the creation of the config!

Thanks to Thunder33345 and TB for helping me fix line 40

### CODE REVIEW

File: src/ProxyBlocker/ProxyChecker.php [hunk 1]
```diff
@@ -1,36 +1,44 @@
...
+    public function onCompletion(Server $server)
     {
         if (($res = $this->getResult())[0]) {
             $server->getPlayerExact($this->data[1])->kick($res[1], false);
             foreach (array_keys($server->getOps()->getAll()) as $op) {
-                if ($pl = $server->getPlayer($op)) instanceof \pocketmine\Player) {
-                    $pl->sendMessage($res[2]);
+                $pl = $server->getPlayer($op);
+                if ($pl instanceof Player) {
+                    {
```
Comment (Muqsit) [inline src/ProxyBlocker/ProxyChecker.php]: unneeded bracket.

File: src/ProxyBlocker/Main.php [hunk 2]
```diff
@@ -2,12 +2,21 @@
...
 
     private $server, $email;
+
     public function onEnable()
     {
+        if(!file_exists($this->getDataFolder() . "config.yml")) {
+            @mkdir($this->getDataFolder());
+            file_put_contents($this->getDataFolder() . "config.yml", $this->getResource("config.yml"));
+        }
+        $this->saveDefaultConfig();
```
Comment (Muqsit) [inline src/ProxyBlocker/Main.php:17]: this is useless

File: src/ProxyBlocker/ProxyChecker.php [hunk 3]
```diff
@@ -1,36 +1,44 @@
...
 namespace ProxyBlocker;
 
+use pocketmine\Player;
+use pocketmine\scheduler\AsyncTask;
+use pocketmine\Server;
 use pocketmine\utils\TextFormat as TF;
 
-class VPNCheckTask extends \pocketmine\scheduler\AsyncTask {
+class ProxyChecker extends AsyncTask
+{
```
Comment (Muqsit) [inline src/ProxyBlocker/ProxyChecker.php]: bad formatting

File: src/ProxyBlocker/Main.php (hunk 2 above)
Comment (Matthww) [reply src/ProxyBlocker/Main.php:17]: Just this line (18) or everything?

**Pull request merged**